hr_employee_exemption/i18n/fr.po

```diff
@@ -0,0 +1,89 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* hr_employee_exemption
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-05-14 14:14+0000\n"
+"PO-Revision-Date: 2015-05-14 10:16-0500\n"
+"Last-Translator: David Dufresne <david.dufresne@savoirfairelinux.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: hr_employee_exemption
+#: model:ir.model,name:hr_employee_exemption.model_hr_salary_rule
+msgid "hr.salary.rule"
+msgstr "hr.salary.rule"
+
+#. module: hr_employee_exemption
+#: field:hr.employee.exemption,exemption_id:0
+#: field:hr.salary.rule,exemption_id:0
+msgid "Exemption"
+msgstr "Exonération"
+
+#. module: hr_employee_exemption
+#: field:hr.income.tax.exemption,name:0
+msgid "Name"
+msgstr "Nom"
+
+#. module: hr_employee_exemption
+#: field:hr.employee,exemption_ids:0 view:hr.income.tax.exemption:0
+#: model:ir.actions.act_window,name:hr_employee_exemption.action_hr_income_tax_exemption_list
+#: model:ir.ui.menu,name:hr_employee_exemption.menu_action_hr_income_tax_exemption_list
+msgid "Income Tax Exemptions"
+msgstr "Exonérations d'impôt"
+
+#. module: hr_employee_exemption
+#: view:hr.employee:0
+msgid "Tax Exemptions"
+msgstr "Exonérations d'impôt"
+
+#. module: hr_employee_exemption
+#: model:ir.model,name:hr_employee_exemption.model_hr_employee_exemption
+msgid "Employee Income Tax Exemption"
+msgstr "Exonération d'impôt de l'employé"
+
+#. module: hr_employee_exemption
+#: view:hr.income.tax.exemption:0
+#: field:hr.income.tax.exemption,salary_rule_ids:0
+msgid "Salary Rules"
+msgstr "Règles de salaire"
+
+#. module: hr_employee_exemption
+#: field:hr.employee.exemption,date_to:0
+msgid "Date To"
+msgstr "Date de début"
```

Comment:
I made this mistake in v7. I wrote Date de début instead of Date de fin. You may fix it.